src/config.rs

```diff
@@ -66,6 +66,9 @@ pub struct Config {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     default_ipv6_prefix: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    is_internal: Option<bool>,
```

Comment:
Please add a doc comment to explain the effect of this config. (Also, please write a descriptive commit message to explain the same thing).

Thank you!

Reply:
Sure, could you review the doc comment before uploading a commit?

```rs
/// This flag indicates that you are an internal user. Validated when using
/// internal functionality. It is false by default.
```

Reply:
> Validated when using internal features

I would say...

> When the config option is true, the behavior of this tool will be optimized for the internal users by default (e.g. using internal manifest for checking out source code by default).

Also, maybe `the flag` should be replaced by `the config option` or `the option`, since it's an option of the config, controlled by `lium config` command

Reply:
Thank you for your advice!!  
I added the doc comment and updated the commit message.